api/app/attendance/api.py

```diff
@@ -106,12 +106,12 @@ def post(self):
 
         send_mail(
             recipient=user.email,
-            subject='Welcome to {}'.format(event.name),
+            subject='Welcome to {}'.format(event.get_name('en')),

```

Comment:
@avishkar58 So you'll see this a lot where I default to English. We do seem to have context of the user in a lot of these cases but I don't think we allow them specify their preferred language (and that's also assuming their preferred language overlaps with the event languages).
Let me know what you think on how to handle this going forward.

Reply:
Hey Amrit, so we do have their preferred language stored in the user_primaryLanguage parameter - but don't worry to change this now because my email templates PR centralises this so I can do it there :)

Reply:
I saw the email translations PR so my question might be resolved there